src/material-experimental/mdc-chips/chip.ts

```diff
@@ -44,7 +44,7 @@ import {
   RippleRenderer,
   RippleTarget,
 } from '@angular/material/core';
-import {MDCChipAdapter, MDCChipFoundation} from '@material/chips';
+import {MDCChipAdapter, MDCChipFoundation} from '@material/chips/chip';
```

Comment:
This workaround is a bit of a code smell, but I don't know whether it's with Bazel or MDC. After the latest update I got a bunch of compilation errors like this for properties that exist:

```
src/material-experimental/mdc-chips/chip.ts:372:32 - error TS2339: Property 'handleTrailingIconInteraction' does not exist on type 'MDCChipFoundation'.

372           this._chipFoundation.handleTrailingIconInteraction(event);
                                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

I've confirmed that these properties exist in the `node_modules` and vscode picks them up correctly, but for some reason running it through Bazel throws the error. I tried debugging it a bit and it seems like a `keyof MDCChipFoundation` resolves to this `"export" | "default" | "MDCChipFoundation" | "init" | "destroy"`. The weird part is that all other MDC imports worked fine and I don't see anything in `chip.ts` that stands out.